crates/sui-rosetta/tests/end_to_end_tests.rs

```diff
@@ -123,7 +123,9 @@ async fn test_get_staked_sui() {
     println!("{}", serde_json::to_string_pretty(&response).unwrap());
 }
 
+// Fails with GasBudgetTooHigh, disabled for now
```

Comment:
3 tests run into GasBudgetTooHigh failure. Spoke with @patrickkuo  offline and decided to disable then for now